_posts/2017-11-11-environments.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Managing software versioning using Conda environments
+---
+
+Research projects can often take months to years to complete, however the
+precise version of software they use will often have a shorter lifetime than
+this.  These new versions of software will often include new features which
+might be of great use, but they might also introduce changes which break
+your existing work and introduce compatibility issues with other pieces
+of software.  So whilst for existing projects we might wish to freeze
+all pieces of software installed, for newer projects we instead want to
+use the most up-to-date versions, leaving us needing to install multiple
+versions of multiple different pieces of software.
+
+In this post we will try to explain how conda and Python virtual environments
+can be used to precisely manage the software used across many independent
+research projects.
+
+
+# Conda Environments
+
+[Conda](https://conda.io/docs/index.html) is a general package manager for
+scientific applications. It is mostly used for Python packages but the system
+can be used with any programs. The [conda-forge](https://conda-forge.github.io/)
+community also provides a large collection of scientific software for Python, R
+and perl. Conda should be your first choice to manage different software
```

Comment:
I don't like the "Conda should be your first choice". It seems very paternalistic. Rather something like "Conda has shown itself to be an excellent tool to manage different software versions." or "Conda is now widely used to manage different software versions." Let your readers decide for themselves if they want to use it.